public/friendlycode/js/fc/filesystem-sync.js

```diff
@@ -0,0 +1,104 @@
+define(["jquery"], function($) {
+  "use strict";
+
+  var FileSystemSync = {};
+
+  function pushFileChange(url, csrfToken, fs, path) {
```

Comment:
Because `Bramble.getFileSystem()` is globally available, maybe you don't need to bother passing `fs` around all the time?